test/e2e/tests/02-blank-form-fails.js

```diff
@@ -0,0 +1,88 @@
+const request = require('request');
+
+const report = require('../report');
+const config = require('../../../config');
+
+// FIXME: This should come in from a config variable
+const url = 'http://localhost:' + config.port + '/healthcare/apply/application/review-and-submit'; // eslint-disable-line
+
+// Test timeout constants
+// TODO(awong): Move to a common file usable for all tests.
+const timeouts = {
+  normal: 500,     // The normal timeout to use. For most opreations w/o a server roundtrip, this should be more than fast enough.
+  slow: 1000,      // A slow timeout incase the page is doing something complex.
+  molasses: 5000,  // A really really slow timeout. This should rarely be used.
+  submission: 10000 // Only to be used for submission.
+};
+
+// TODO(awong): Move this into a custom command or assertion that can be used with client.expect.element().
+function expectNavigateAwayFrom(client, urlSubstring) {
+  client.expect.element('.js-test-location').attribute('data-location')
+    .to.not.contain(urlSubstring).before(timeouts.normal);
+}
+
+const section = [
+  'confirm-personal-information',
+  'confirm-birth-information',
+  'confirm-demographic-information',
+  'confirm-veteran-address',
+  'confirm-contact-information',
+  'confirm-military-service-information',
+  'confirm-additional-service-information',
+  'confirm-basic-information',
+  'confirm-financial-disclosure',
+  'confirm-spouse-information',
+  'confirm-child-information',
+  'confirm-income-information',
+  'confirm-deductible-expenses',
+  'confirm-medicare-medicaid-information',
+  'confirm-insurance-information',
+  'confirm-additional-information'
+];
+
+module.exports = {
+  'Begin application': (client) => {
+    console.log(url);
+    request({
+      uri: `${url}/api/hca/v1/mock`,
+      method: 'POST',
+      json: {
+        resource: 'application',
+        verb: 'post',
+        value: {
+          formSubmissionId: '123fake-submission-id-567',
+          timeStamp: '2016-05-16'
+        }
+      }
+    });
+
+    // Review and Submit Page.
+    client
+      .url(url)
+      .waitForElementVisible('body', timeouts.normal)
+      .assert.title('Apply for Health Care: Vets.gov')
+      .waitForElementVisible('.form-panel', timeouts.slow);  // First render of React may be slow.
+
+    // confirm all sections (even though they're blank)
+    for (var i = 0; i < section.length; i++) { // eslint-disable-line
```

Comment:
I'm a little confused by what this is doing. I think you can submit the form without updating each of the sections or clicking that you verify anything. In fact, if you click update for a blank section, it shouldn't allow you to proceed to the next one because it is invalid. Is this necessary to do for the purpose of the test?